packages/unity-react-core/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {useEffect} from "react";
+/**
+ *
+ * TODO: Should we be using bootstrap's built in modal functionality?
+ */
+
+export const Modal: React.FC = () => {
+
+  useEffect(() => {
+      document
+        ?.getElementById('openModalButton')
+        .addEventListener('click', function () {
+          document.getElementById('uds-modal').classList.add('open');
+        });
+
+      document
+        ?.getElementById('closeModalButton')
+        .addEventListener('click', function () {
+          document.getElementById('uds-modal').classList.remove('open');
+        });
+  }, []);
+
+  return (
+    <div className="container-fluid">
+      <button id="openModalButton" className="btn btn-dark">
```

Comment:
I know this is example code only so I would like to capture the direction we want to move in. Can you add these notes as comments (if you agree) to the component?
 - I'd like the Modal to only be the content of the modal. (without the button).
 - React 18 has createPortal. Maybe we use this so the modal can be attached to the bottom of the body?
 - We should be able to use the default button component to trigger the modal,
 - We should also be able to trigger the modal on other events,